org/mozilla/jss/nss/SSL.c

```diff
@@ -278,6 +278,66 @@ Java_org_mozilla_jss_nss_SSL_VersionRangeGet(JNIEnv *env, jclass clazz,
     return JSS_SSL_wrapVersionRange(env, vrange);
 }
 
+JNIEXPORT int JNICALL
+Java_org_mozilla_jss_nss_SSL_VersionRangeSetDefaultNative(JNIEnv *env, jclass clazz,
+    jint variant_ssl, jint min_ssl, jint max_ssl)
+{
+    SSLVersionRange vrange;
+    SSLProtocolVariant variant;
+
+    PR_ASSERT(env != NULL);
+    PR_SetError(0, 0);
+
+    if (min_ssl < 0 || min_ssl >= JSSL_enums_size ||
+            max_ssl < 0 || max_ssl >= JSSL_enums_size ||
+            variant_ssl < 0 || variant_ssl >= JSSL_enums_size)
+    {
+        char buf[148];
+        snprintf(buf, 148,
```

Comment:
Is this the exact length of the current string? I'd give it a larger size (e.g. 200) to accommodate the variables and the possibilities of someone updating the text without updating the buffer size.

Reply:
The length of the string is 108 without digit placeholders. Each number can be at most 11 digits (since it is a 32-bit signed integer), which means the result fits in 108 + 4*11 = 152 characters. So when numbers were negative, it was actually too small!

I've fixed this to 200 as suggested. However, note that this is unlikely, as it mostly connotes a mismatch between JSS Java and JSS Native components. (Or reflection).